setup.py

```diff
@@ -67,7 +66,11 @@ def package_files(directory):
         "console-progressbar==1.1.2",
         "ansicolors==1.1.8",
         "pluggy",
-        "qtpy"
+        "qtpy",
+        "pyyaml",
+        "sphinx",
+        "sphinx-argparse",
+        "sphinx_rtd_theme",
```

Comment:
I vote that we move these requirements to a separate `requirements-dev.txt` (or something similar), along with the test-dependencies.

Reply:
I don't disagree with that, there seems to be no clear consensus about how to do this in the Python community, but seems to be a pretty common solution.